Kconfig.zephyr

```diff
@@ -648,12 +648,13 @@ config OUTPUT_SYMBOLS
 
 config OUTPUT_DISASSEMBLY
 	bool "Create a disassembly file"
+	default y
 	help
 	  Create an .lst file with the assembly listing of the firmware.
 
 config OUTPUT_DISASSEMBLE_ALL
 	bool "Disassemble all sections with source. Fill zeros."
-	default n
+	default y
```

Comment:
this will affect all boards in Zephyr, please do this only for the board (or SoC) where the default should be changed, or provide a better reason why this change should be accepted.